frontend/src/Values/locales/fr/translation.json

```diff
@@ -124,8 +124,7 @@
   "footer": {
     "connectList": {
       "title": "Connect",
-      "address": "5678 Power Rd South Gloucester Industrial Park",
-      "email": "{Insert Email}",
+      "address": "5678 Power Rd",
```

Comment:
Should this be power road south like the previous ones?

Reply:
Good catch, resolved.